### DISCUSSION TITLE
Question about index and currency parameters in Item

### DISCUSSION THREAD

**Discussion by abulenok** [invertase/react-native-firebase#6386] [Q&A]
Hello,  I wanted to ask is there is some reason why `index` and `currency` parameters are not added to the Item interface 

https://github.com/invertase/react-native-firebase/blob/72401288eb7cbfe3246ffaa384985135a74c929a/packages/analytics/lib/index.d.ts#L53 

Do you plan to add them in the future?

Item parameters in GA4 docs https://developers.google.com/analytics/devguides/collection/ga4/reference/events#purchase_item

**Comment by mikehardy** [comment]
No real answer, sorry - other than perhaps, I'm not sure? Could be an oversight. If you see the object shape of Item matching the doc you quote almost completely but missing those two, you could post a PR to add them and we could merge it I think?
